resources/geocoding/hy/374.txt

```diff
@@ -59,11 +59,11 @@
 37423492|Մարտիրոսյան
 37423497|Փոքր Վեդի
 37423498|Տափերական
-374235|Արտաշատ/Այգեզարդ/Դալար/Քաղցրաշեն/Մխչյան/Շահումյան
+374235|Արտաշատ/Այգեվազ/Դալար/Քաղցրաշեն/Մխչյան/Շահումյան
 37423572|Արևշատ
 3742359|Նորաշեն
-374236|Այնթապ/Մասիս/Նոր Խարբերդ/Նորաբաց
-3742363|Այնթապ
+374236|Այնթափ/Մասիս/Նոր Խարբերդ/Նորաբաց
```

Comment:
Google Search results seem to suggest that Այնթապ is more common (I get 180,000 results for Այնթապ vs. 60,000 for Այնթափ). Also I'm personally only familiar with Այնթապ :) This is also the name in Maps: https://goo.gl/maps/oGtt5UCLsXo